opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/QueryQuery.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.opensearch.storage.script.filter.lucene.relevance;
+
+import org.opensearch.index.query.QueryBuilders;
+import org.opensearch.index.query.QueryStringQueryBuilder;
+
+/**
+ * Class for Lucene query that builds the query_string query.
+ */
+public class QueryQuery extends NoFieldQuery<QueryStringQueryBuilder> {
+
+  final String queryQueryName = "query";
```

Comment:
make it `private` or even a constant?

Reply:
Addressed in https://github.com/opensearch-project/sql/pull/1010/commits/e73cf7dd5f21f51f250857fa71fc29eb3d1894f0